### PR TITLE
[Inductor] [CI] [CUDA] Skip the failed models and tests the better way

### PR DESCRIPTION
**Pull request by nWEIdia**
Address subtasks in https://github.com/pytorch/pytorch/issues/126692 

After enabling the disabled shards, the following two models regressed (for cu124 configuration): 
dynamic_inductor_timm_training.csv  
cspdarknet53,pass,7   (expected)                                        | cspdarknet53,fail_accuracy,7           (actual)
eca_botnext26ts_256,pass,7        (expected)                            | eca_botnext26ts_256,fail_accuracy,7 (actual)

cc @albanD @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @atalman @malfet @ptrblck @eqy @tinglvv @Aidyn-A

### CODE REVIEW

**Comment by huydhn** [comment]
@nWEIdia I should have pinged you earlier, but I have a small fix for a 12.4/12.1 typo on inductor workflow here https://github.com/pytorch/pytorch/pull/127121 that has just been landed.  You might want to merge in that change.

**Review by huydhn — APPROVED** [review]

File: benchmarks/dynamo/ci_expected_accuracy/dynamic_inductor_timm_training.csv [hunk 1]
```diff
@@ -6,7 +6,7 @@ adv_inception_v3,pass,6
 
 
 
-beit_base_patch16_224,pass,7
+beit_base_patch16_224,fail_accuracy,7
```
Comment (huydhn) [inline benchmarks/dynamo/ci_expected_accuracy/dynamic_inductor_timm_training.csv:9]: I realize that there is no clear way to tell tat this is failing accuracy only on 12.4?

Comment (nWEIdia) [reply benchmarks/dynamo/ci_expected_accuracy/dynamic_inductor_timm_training.csv:9]: You are right. I am still trying to figure out how to branch as CUDA 12.1 would pass...

**Comment by nWEIdia** [comment]
> @nWEIdia I should have pinged you earlier but missed your PR. I have a small fix for a 12.4/12.1 typo on inductor workflow here #127121 that has just been landed. You might want to merge in that change.

Thank you for the fix!

File: .ci/pytorch/test.sh [hunk 2]
```diff
@@ -589,7 +589,8 @@ test_inductor_torchbench_smoketest_perf() {
   # https://github.com/pytorch/pytorch/actions/runs/7158691360/job/19491437314,
   # and thus we lower its threshold to reduce flakiness. If this continues to be a problem,
   # we switch to use some other model.
-  python benchmarks/dynamo/check_perf_csv.py -f "$TEST_REPORTS_DIR/inductor_inference_smoketest.csv" -t 4.9
+  # Change 4.9 to 4.7, change back after fixing https://github.com/pytorch/pytorch/issues/126692
+  python benchmarks/dynamo/check_perf_csv.py -f "$TEST_REPORTS_DIR/inductor_inference_smoketest.csv" -t 4.7
```
Comment (nWEIdia) [inline .ci/pytorch/test.sh:593]: And here we probably need to if else for cuda 12.1 and cuda 12.4 too..

**Review by eqy — APPROVED** [review]

File: benchmarks/dynamo/ci_expected_accuracy/dynamic_inductor_timm_training.csv (hunk 1 above)
Comment (nWEIdia) [reply benchmarks/dynamo/ci_expected_accuracy/dynamic_inductor_timm_training.csv]: Duplicated the csv files to a cu124 directory and it seems to work.

File: .ci/pytorch/test.sh (hunk 2 above)
Comment (nWEIdia) [reply .ci/pytorch/test.sh]: Did if else switch to use 4.9 for cu121 and 4.7 for cu124

**Review by atalman — APPROVED** [review]
lgtm

**Pull request closed without merging**